src/Components/Aspire.Azure.Npgsql/api/Aspire.Azure.Npgsql.cs

```diff
@@ -0,0 +1,25 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+namespace Aspire.Azure.Npgsql
```

Comment:
@eerhardt @sebastienros @davidfowl This new package is also set to ship as stable for 9.2. Is that intentional?

Reply:
I think it should be?

Reply:
Ok, it does expose very minimal API and it is just following precedent, but wanted to check in case we wanted to give it a bit more bake time.